doc/05_Objects/01_Object_Classes/03_Layout_Elements/01_Dynamic_Text_Labels.md

```diff
@@ -24,18 +24,18 @@ Here is an example for a rendering class.
 
 namespace App\Helpers;
 
-use Pimcore\Model\DataObject\Concrete;
+use Pimcore\Model\DataObject\ClassDefinition\Layout\DynamicTextLabelInterface;
 
 class CustomRenderer implements DynamicTextLabelInterface
 {
     /**
      * @param string $data as provided in the class definition
      */
-    public function renderLayoutText(string $data, ?Concrete $object, array $params): string
+    public function renderLayoutText($data, $object, $params): string
```

Comment:
@pdchaudhary The interface has type hints, see: https://github.com/pimcore/pimcore/blob/11.x/models/DataObject/ClassDefinition/Layout/DynamicTextLabelInterface.php#L27

Reply:
You are right, I am using an old version. :)